### ISSUE TITLE
Creare un prototipo di oggetto di scena

### ISSUE THREAD

**Issue by wbigger** [marconicivitavecchia/the-school-of-athens-vr#32]
Partendo dalla lista degli oggetti di scena, selezionarne uno e farne un prototipo che poi potrà essere utilizzato anche come esempio per gli altri oggetti.

**Comment by wbigger** [comment]
Potete trovare la lista dei props da sviluppare [qui](https://github.com/marconicivitavecchia/the-school-of-athens-vr/tree/main/props)

Prima di passare all'implementazione vera e propria, sarà necessario fare uno sketch 2D dello stesso oggetto, per studiarne bene le proprietà. Se serve un aiuto a riguardo, @Andrea-Arilli @DarioTomei fate sapere.

**Comment by Andrea-Arilli** [comment]
![Image](https://user-images.githubusercontent.com/126103544/226615888-71c6c24b-70c7-46f5-9586-43084a29e359.png)



![Image](https://user-images.githubusercontent.com/126103544/226615906-95a07a61-228b-4465-977e-624e7aa9956e.png)

**Comment by wbigger** [comment]
@Andrea-Arilli @DarioTomei è importante che condividiate anche il file .blend!

Non potete allegarlo direttamente qui (in queste discussioni si possono allegare solo immagini, pdf e qualche altro formato), ma potete caricarlo direttamente sul progetto GitHub.

Fate così: andarte sul progetto [GitHub](https://github.com/marconicivitavecchia/the-school-of-athens-vr), selezionate il branch "props" (vedi immagine)

![Image](https://user-images.githubusercontent.com/3594326/227713909-fe23e2b9-91c5-438e-92ad-ef11c7b3278a.png)

Quindi selezionate la cartella "props"


![Image](https://user-images.githubusercontent.com/3594326/227713956-ed6125ac-0fc2-45f4-b386-3a56c1e4d1a8.png)

Cliccate su Add File

![Image](https://user-images.githubusercontent.com/3594326/227713974-ddf69a85-34e8-443f-8eea-80eaaea580bf.png)

e aggiungete i file .blend

Poi avvisate qui quando avete caricato i file, così li possiamo andare a scaricare!!

**Comment by Andrea-Arilli** [comment]
prof buongiorno, ho appena inserito i file dei props su dove lei aveva detto quindi può andare a vederli

**Comment by wbigger** [comment]
Bene! Per comodità metto il link anche [qui](https://github.com/marconicivitavecchia/the-school-of-athens-vr/tree/props/props).

@DavideSky @Prinxino se avete tempo per mercoledì gli date un'occhiata?

**Comment by DavideSky** [comment]
@wbigger Ho visto entrambi i modelli e questo è quello che ho notato:
### Lira di Apollo

- Ci sono alcune facce orientate al contrario per via di alcuni punti posizionati in modo improprio (si capisce meglio dalla prima immagine)

![image](https://user-images.githubusercontent.com/90967735/227786116-1dab1df9-a94f-4298-8404-dba0ffc393b8.png)
![image](https://user-images.githubusercontent.com/90967735/227785973-f554e729-7432-4c99-b004-1e0e3cbea83b.png)

- La posizione dell'origine dell'oggetto dovrebbe essere al momento dell'esportazione 0,0,0 per facilitare il posizionamento sul motore di gioco. È comunque una piccolezza facilmente risolvibile

![image](https://user-images.githubusercontent.com/90967735/227786376-4f5f3e42-16c8-42f1-a772-0a42c934a9e7.png)

- La geometria delle corde della lira interseca la geometria del corpo principale dello strumento, non so se potrebbe creare dei problemi

![image](https://user-images.githubusercontent.com/90967735/227786904-6ef078d7-020b-473c-a69f-e4f90ce11ef6.png)

-------------------------------------------------

### Libro di Aristotele
- Anche qui ci sono diverse facce orientate al contrario (parti rosse nella foto)

![image](https://user-images.githubusercontent.com/90967735/227786481-0121ef71-e04a-48a6-b376-70464cc4e865.png)

- L'origine non è nelle coordinate 0,0,0 (per lo stesso motivo della lira di Apollo)

![image](https://user-images.githubusercontent.com/90967735/227786618-a5b82de4-c9e4-431a-b190-b9902a2d594c.png)

- Ci sono dei punti non manifold

![image](https://user-images.githubusercontent.com/90967735/227786716-2644905e-57b2-4134-bfa4-c62b6998c36c.png)

- C'è un problema con queste due parti del libro (vedi immagine sotto), è come se ci fossero due facce in sovrapposizione.

![image](https://user-images.githubusercontent.com/90967735/227787251-eff7a288-0141-429e-8eab-06d3d4b1d77c.png)

-----------------------------------------------

Per ora questo è quello che ho notato, per quanto riguarda la densità poligonale penso che ci siamo, bisogna comunque vedere la performance sull'Oculus, ma non credo ci siano troppi problemi

**Comment by wbigger** [comment]
Grazie @DavideSky !
@Andrea-Arilli @DarioTomei riuscite a risolvere questi problemi?

**Comment by wbigger** [comment]
Credo sia a questo punto utile ufficializzare una checklist:
- [ ] origine in (0,0)
- [ ] orientamento coerente delle facce
- [ ] punti non-manifold
- [ ] altro?

Per le intersezioni, @DavideSky facci sapere se è un problema oppure può andar bene così.

**Comment by wbigger** [comment]
Il problema delle facce sovrapposte nel libro dovrebbe essere una conseguenza dei punti non-manifold, probabilmente le due cose si risolvono insieme.

**Comment by DavideSky** [comment]
> 

Esatto, risolti i punti non manifold anche le altre cose dovrebbero risolversi
